### PR TITLE
Feature/asep 33: Implemented geospatial queries for POI API

### PR DESCRIPTION
**Pull request by niels89**
This PR includes the following changes:

- POI API: FeatureRepository uses geospatial query to select pois in radius
- Swagger UI is added, can be accessed via http://localhost:<port>/docs/ui/swagger, so if you use the composed setup, you can browse: http://localhost:1337/docs/ui/index.html
- Adapted build script

### CODE REVIEW

**Review by paluchs — COMMENTED** [review]
Looks good! Do I understand it correctly that you overload getFeatures? Is it necessary to have getFeatures without a radius?

**Comment by niels89** [comment]
> 
> 
> Looks good! Do I understand it correctly that you overload getFeatures? Is it necessary to have getFeatures without a radius?

You're absolutely right, that's a leftover from the first version. I'll remove it and then merge the PR.

File: eloquent-elks-poi-api/src/main/java/eloquentelks/poi/api/Application.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;
 
 @EnableMongoRepositories(basePackageClasses = FeatureRepository.class)
-@SpringBootApplication
+@SpringBootApplication()
```
Comment (saurabhprasun20) [inline eloquent-elks-poi-api/src/main/java/eloquentelks/poi/api/Application.java:9]: Does () add anything in annotation here?

**Review by saurabhprasun20 — APPROVED** [review]

File: eloquent-elks-poi-api/src/main/java/eloquentelks/poi/api/Application.java (hunk 1 above)
Comment (niels89) [reply eloquent-elks-poi-api/src/main/java/eloquentelks/poi/api/Application.java:9]: Well spotted, this is a leftover from something I tried. I will remove it.

**Pull request merged**